rfcs/text/0000-roadmaps+planning.md

```diff
@@ -0,0 +1,421 @@
+
+**Project Leads**, **Product Owners**, and **Technical Leads** are roles identified in DCP [charters](https://github.com/HumanCellAtlas/dcp-community/tree/master/charters) with further details available in the [PM](https://github.com/HumanCellAtlas/dcp-community/blob/master/charters/PM/charter.md) and [Architecture](https://github.com/HumanCellAtlas/dcp-community/blob/master/charters/Architecture/charter.md) charters.
+
+**Themes** represent a discrete user or stakeholder need that delivers on the long term vision of the HCA. A Theme motivates a collection of work performed by the DCP.
+
+**Roadmap Objectives** are the concrete elements of an overarching Theme.
+
+**Product Roadmaps** identify and express user needs modeled as high-level Themes that capture user priorities. Roadmaps align the DCP community around a common set of Themes and get scientists excited about future releases.
+
+**Science Roadmaps** communicate a high-level vision of the goals of the single-cell community and a trajectory of progress towards those goals. It identifies both use cases for the constructed Atlas and open scientific questions that must be solved to build it.
+
+**[User Stories](http://agiledictionary.com/277/user-story/)** are requirements "stated as a sentence or two of plain English. A user story is often expressed from the user’s point of view, and describes a unit of desired functionality."
+
+**[Epics](http://agiledictionary.com/309/epic/)** 
+are a collection of User Stories required to complete an objective.
+
+**[Spikes](http://agiledictionary.com/209/spike/)** are adopted from eXtreme Programming for activities that require research or design before estimation and final implementation can occur.
+- [Example use in the Scaled Agile Framework](https://www.scaledagileframework.com/spikes/)
+
+**Milestones** represent a four week timeline/duration in a Release.
+
+**Releases** are composed of three milestones and occur on a quarterly cadence.
+
+The **Product Backlog** is the set of User Stories and Spikes scheduled for the next Release based on Roadmap priorities. The **Engineering Plan** is another term for the Product Backlog, and is used interchangeably.
+
+## Detailed Design for Roadmaps
+
+The Product Roadmap creates a shared understanding across the DCP community and Users of the DCP for investments and direction for the next year. The Roadmap process MUST occur on an annual basis. 
+
+Themes and Objectives from the Roadmap are the primary source of priorities for planning Releases each quarter.
+
+### Timebox for Roadmaps
+
+All timeboxes are noted as business days. Business days are included regardless of national holidays.
+
+Overlapping timeboxes denote concurrent work.
+
+Time boxes with a range of days indicate that the work MAY be completed at any point during the timebox, but MUST be completed no later than the final day of the timebox.
```

Comment:
MAY is not good enough. Has to be MUST otherwise it will not happen among the sea of other work that rech-arch needs to do. Also needs to be explicitly in the timeline.

Reply:
The MAY indicates that it can be completely earlier like for _extra credit_. But it still MUST be completed on time.  I don't see the problem here.

Reply:
Sorry attached this to the wrong bit.